.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
```

Comment:
Consider using a specific version tag instead of `./` for the action. This ensures reproducibility and prevents unexpected changes.